components/snowflake/actions/execute-sql-query/execute-sql-query.mjs

```diff
@@ -0,0 +1,28 @@
+import snowflake from "../../snowflake.app.mjs";
+
+export default {
+  name: "Execute SQL Query",
```

Comment:
**Nitpick**
Snowflake is not technically SQL, perhaps we should name it like this?
```suggestion
  name: "Execute Query",
```